release-scripts/utils.py

```diff
@@ -29,6 +29,9 @@
 VERSION_RE = re.compile(r'^v(\d+)\.(\d+)\.(\d+)$')
 VERSION_NAME_RE = re.compile(r'^v(\d+)\.(\d+)\.(\d+)[.-](\w+)$')
 
+# Regex for MD file URL matching
+MD_URL_RE = re.compile(r'\[([^\[\]]*)\]\(([^()]*)\)')
```

Comment:
Ha!  Its not that bad, it's always a pain if you need to match on []() - looks worse than it really is